scripts/npn.py

```diff
@@ -51,7 +54,7 @@ def download(self, engine=None, debug=False):
             engine.create_table()
 
             base_url = 'http://www.usanpn.org/getObs/observations/'
-            years = range(2009, 2013)
+            years = list(range(2009, 2013))
 
```

Comment:
@goelakash, so we have wrapped range(2009, 2013) -> list(range(2009, 2013))
and if this change is relevant to py3 we can go ahead and wrap other similar occurrences of range.
What do you think?